integration/mediation-tests/tests-mediator-1/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/proxy-services/MultipleOutMediatorsTestCaseProxy.xml

```diff
@@ -0,0 +1,40 @@
+<proxy xmlns="http://ws.apache.org/ns/synapse" name="MultipleOutMediatorsTestCaseProxy" startOnLoad="true" statistics="disable" trace="disable" transports="http,https">
```

Comment:
shall we add licence header

Reply:
Added. Please find the changes in the recent commit